src/main/webapp/WEB-INF/tags/instructor/course/studentTableModal.tag

```diff
@@ -4,13 +4,13 @@
   <div class="modal-dialog modal-lg">
     <div class="modal-content">
       <div class="modal-header">
-        <span class="pull-left help-block">
-          Tips: After Selecting the table, <kbd>Ctrl + C</kbd> to COPY and <kbd>Ctrl + V</kbd> to PASTE to your Excel Workbook.
-        </span>
-
-        <button type="button" class="btn btn-default" data-dismiss="modal">
-          Close
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
         </button>
+        <span class="help-block">
+          Tips: After selecting the table, <kbd>Ctrl + C</kbd> or <kbd>⌘ + C</kbd> to COPY and
+          <kbd>Ctrl + V</kbd> or <kbd>⌘ + V</kbd> to PASTE to your Excel Workbook.
+        </span>
 
         <button type="button" class="btn btn-primary" id="btn-select-element-contents">
```

Comment:
Can we use a link instead of a button, as in my original suggestion?
<img width="908" alt="screen shot 2017-10-27 at 4 39 03 pm" src="https://user-images.githubusercontent.com/6811428/32095616-504b32c0-bb36-11e7-93c8-38283861c722.png">

The content is already automatically selected, so we don't even expect the user to use this option under normal circumstances. A link would suffice. Just do `<span class="btn-link" id="btn-select-element-contents">selecting</span>`